src/functionalTest/resources/application-functional.yaml

```diff
@@ -18,10 +18,7 @@ s2s-secret: ${S2S_SECRET:K6KA6PRLRFWQLSD6}
 
 spring.flyway.enabled: false
 ## Secure roles configuration 1
-exui.role.hmcts-admin: prd-admin
-exui.role.pui-user-manager: pui-user-manager
+exui.role.caseworker: caseworker
 exui.role.pui-organisation-manager: pui-organisation-manager
```

Comment:
do we need exui role configuration for judicial api?

Reply:
ok

Reply:
kept one role for negative testing , that's ok